trieste/acquisition/function/function.py

```diff
@@ -762,3 +763,74 @@ def acquisition(x: TensorType) -> TensorType:
         return tf.exp(tf.linalg.logdet(covariance + jitter))
 
     return acquisition
+
+
+class BALD(SingleModelAcquisitionBuilder):
+    """
+    Builder for the *Bayesian Active Learning By Disagreement* acquisition function defined in
+    :cite:`houlsby2011bayesian`. The acquisition function computes the information gains of the predictive entropy.
+
+    This acquisition function is intended to use for Binary Gaussian Process Classification model with Bernoulli likelihood.
+    """
+
+    def __init__(self, jitter: float = DEFAULTS.JITTER) -> None:
+        """
+        :param jitter: The size of the jitter to avoid numerical problem caused by the variance value.
+        """
+        self._jitter = jitter
+
+    def __repr__(self) -> str:
+        """"""
+        return f"BALD(jitter={self._jitter!r})"
+
+    def prepare_acquisition_function(
+        self,
+        model: ProbabilisticModel,
+        dataset: Optional[Dataset] = None,
+    ) -> AcquisitionFunction:
+        """
+        :param model: The model.
+        :param dataset: Unused.
+
+        :return: The determinant of the predictive function.
+        """
+
+        return bald(model, self._jitter)
+
+    def update_acquisition_function(
+        self,
+        function: AcquisitionFunction,
+        model: ProbabilisticModel,
+        dataset: Optional[Dataset] = None,
+    ) -> AcquisitionFunction:
+        """
+        :param function: The acquisition function to update.
+        :param model: The model.
+        :param dataset: Unused.
+        """
+        return function  # no need to update anything
+
+
+class bald(AcquisitionFunctionClass):
+    def __init__(self, model: ProbabilisticModel, jitter: float):
+        """
+
+        :param model: The model of the objective function.
+        :param jitter: The size of the jitter to avoid numerical problem caused by the variance value.
```

Comment:
jitter is used below to avoid issue in taking logs of values close to zero, its not really related to variance value

Reply:
indeed that's what I want to say, thanks for the correction. fixed.